app/src/time/timecontrol.cpp

```diff
@@ -65,7 +63,7 @@ std::ostream &operator<<(std::ostream &os, const TimeControl &tc) {
         return os;
     }
 
-    if (tc.limits_.moves == 0 && tc.limits_.time == 0 && tc.limits_.increment == 0) {
+    if (tc.limits_.moves == 0 && tc.limits_.time + tc.limits_.increment == 0) {
```

Comment:
this change isnt needed, time aint negative right ? and so we have 0 + 0 == 0 and before == 0 && .. == 0

Reply:
yeah, this ones just a formatting change to be consistent but ill switch it back if you want

Reply:
switched it back